assets/pages/bundles-landing/bundlesLanding.jsx

```diff
@@ -25,12 +26,14 @@ const participation = pageStartup.start();
 
 
 // ----- Redux Store ----- //
+const intCmp = getQueryParameter('INTCMP', 'gdnwb_copts_bundles_landing_default') || '';
```

Comment:
This will never hit the empty string because we give it `gdnwb_copts_bundles_landing_default` as the default.

Reply:
What do you mean with 'hit'? It should show the other ways of support by default.